### ISSUE TITLE
As data manager I would like to be able to upload a script from file

### ISSUE THREAD

**Issue by fdlk** [molgenis/molgenis-tools-commander#94]
I can already upload an EMX file containing one or more scripts in the sys_scr_Script tab but this format is not very handy for such large files.
It's hard to keep track of changes this way.

So I'd like to be able to upload a script from file and specify the details, like extension and if a token should be generated, as parameters with sensible defaults

**Comment by mswertz** [comment]
I'd rather change MOLGENIS itself to allow upload of actual script files. In my view I should simply be able to upload scripts into a folder so I can see them in the navigator. Then it should "just work"(TM), optionally with some header metadata to pass some magic. Then I can keep my scripts in github repo.

**Comment by fdlk** [comment]
@mswertz I don't think this is either/or. It's fine with me if the navigator supports it as well, but I need it in the commander.

**Comment by dtroelofsprins** [comment]
Not only be able to upload, but also be able to start it with the commander?
